src/pages/CreateEvent/EventForm.tsx

```diff
@@ -194,469 +196,482 @@ const EventForm: React.FC = () => {
   return (
     <form onSubmit={handleSubmit(onSubmit)} className="w-2/3 mx-auto py-[3.5rem] px-[1.5rem] rounded-lg font-gilroy">
       <section className="mb-[1.3rem]">
-        <h2 className="text-black font-bold text-[3rem] leading-[115.645%]">Create an Event</h2>
-        <div className="flex gap-[1rem] my-[3.5rem]">
-          <div className="flex flex-1 gap-[0.8rem] py-[0.6rem] justify-center items-center text-center text-[1.2rem] rounded-[0.6rem] border-t-[0.2rem] border-l-[0.2rem] border-b-[0.3rem] border-r-[0.3rem] border-MVP-black bg-MVP-yellow text-MVP-black font-extrabold"><div className="justify-center items-center rounded-full h-full border-MVP-black border-[0.1rem] aspect-square">1</div> Event Details</div>
-          <div className="flex flex-1 gap-[0.8rem] py-[0.6rem] justify-center items-center text-center text-[1.2rem] rounded-[0.6rem] border-t-[0.2rem] border-l-[0.2rem] border-b-[0.3rem] border-r-[0.3rem] border-MVP-black font-extrabold"><div className="justify-center items-center rounded-full h-full border-MVP-black border-[0.1rem] aspect-square">2</div> Challenge Details</div>
-          <div className="flex flex-1 gap-[0.8rem] py-[0.6rem] justify-center items-center text-center text-[1.2rem] rounded-[0.6rem] border-t-[0.2rem] border-l-[0.2rem] border-b-[0.3rem] border-r-[0.3rem] border-MVP-black font-extrabold"><div className="justify-center items-center rounded-full h-full border-MVP-black border-[0.1rem] aspect-square">3</div> Review</div>
+        <h2 className="font-bold text-[3rem] leading-[115.645%]">Create an Event</h2>
+        <div className="flex gap-[2rem] my-[3.5rem] font-extrabold text-[1.2rem]">
+          <div className={`${styledBorder} rounded-none flex flex-1 gap-[0.8rem] !py-[0.6rem] !px-0 justify-center items-center text-center bg-MVP-yellow`}><div className="flex justify-center items-center rounded-full h-full border-MVP-black border-[0.1rem] aspect-square">1</div> Event Details</div>
+          <div className={`${styledBorder} rounded-none flex flex-1 gap-[0.8rem] !py-[0.6rem] !px-0 justify-center items-center text-center`}><div className="flex justify-center items-center rounded-full h-full border-MVP-black border-[0.1rem] aspect-square">2</div> Challenge Details</div>
+          <div className={`${styledBorder} rounded-none flex flex-1 gap-[0.8rem] !py-[0.6rem] !px-0 justify-center items-center text-center`}><div className="flex justify-center items-center rounded-full h-full border-MVP-black border-[0.1rem] aspect-square">3</div> Review</div>
         </div>
         <div className="py-[0.5rem] px-[2rem]">
-        <div className="mb-[1rem] flex flex-col">
-          <label htmlFor="event title" className="flex items-center gap-[0.2rem] mb-[0.3rem] text-MVP-black text-[1.6rem] font-extrabold">Event Title<span className="mb-2/3 text-[2rem]">*</span></label>
```

Comment:
<img width="901" alt="image" src="https://github.com/user-attachments/assets/69db7bc8-639f-4c49-9444-5ec10e706928">



I see event title has a default value of 'test' now, I am wondering if this is what we are supposed to have.

Reply:
This also happens to some other values, such as the judge name having a default value of "test test".

Reply:
oh i think because of the values are saved to localStorage when you click next, so you can press cancel on this page and it will delete the values. 
It is in the onSubmit function for this page:
 ```
 const onSubmit = (data: EventFormInputs) => {
    const validTheme = validateField("themes", watchedThemes);
    const validDiscipline = validateField("disciplines", watchedDisciplines);
    if (!(validTheme || validDiscipline)) {
      return;
    }
    saveFormData("eventFormData", data);
    navigate("/ChallengeDetails");
  };
```